tasks/jsonlint.js

```diff
@@ -10,10 +10,11 @@ module.exports = function (grunt) {
   "use strict";
 
   var strip = require('strip-json-comments');
-  var jsonlint = require('jsonlint');
+  var jsonlint = require('@prantlf/jsonlint');
+  var sorter = require('@prantlf/jsonlint/lib/sorter');
```

Comment:
In general, I recommend against linking directly into a 3rd party module like this.  `grunt-jsonlint` and `@prantlf/jsonlint` are separate modules.  `@prantif/jsonlint` should export an interface to be developed against.

However, since it's your module you're consuming and there are good tests on the feature, I'm comfortable with it.